packages/fxa-content-server/app/scripts/lib/survey-filter.js

```diff
@@ -350,11 +350,18 @@ export const createSurveyFilter = (
   return async (surveyConfig) =>
     !!(
       surveyConfig &&
-      surveyConfig.rate &&
-      withinRate(surveyConfig.rate) &&
+      parseInt(surveyConfig.rate) >= 0 &&
       surveyConfig.conditions &&
+      withinRate(
+        surveyConfig.conditions.always === true ? 1 : surveyConfig.rate
```

Comment:
Explicitly check `true` so we don't allow other truthy values to accidentally trigger it.

Reply:
I prefer `(surveyConfig.conditions.always === true || withinRate(surveyConfig.rate))` over a ternary.